### DISCUSSION TITLE
Having AI make fun of my failures: Great idea, Git

### DISCUSSION THREAD

**Discussion by jeffshumphreys** [community/community#107492] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I've got the Arctic something achievement. Because I've contributed 0 to some archive.  Very kind.
Now I have the world's fastest closure of an issue.  Not sure, but I'm assuming that's a gag too.  If I can close it in 15 minutes, maybe it aught shuld a been an issue?

Any way to delete all these "Achievements"?

**Comment by ZephyrusBlaze** [comment, marked as answer]
You can't directly delete GitHub achievements or badges, but you can hide them. Go to Settings > Public profile > Profile settings, then uncheck "Show Achievements on my profile" and click on "Update preferences" to hide them from your profile.
![Screenshot_20240211_122439_Brave](https://github.com/community/community/assets/113236396/5cc69f61-a083-4299-acb5-4ecedb154a2c)

**Reply by jeffshumphreys** [reply]
I'd prefer not achieving things, but this is a good solution.
